docs/apache-airflow/public-airflow-api.rst

```diff
@@ -0,0 +1,141 @@
+and data structures used to manage DAGs and tasks, are not considered to be part of the public API.
+These implementation details are not intended to be used by external developers, a
+nd are subject to change without notice.
+
+Additionally, certain features and functionality of Apache Airflow may not be exposed through the public API,
+either because they are not considered to be stable or because they are not intended for external use.
+In these cases, developers may not be able to access these features through the public API,
+and should instead use other available methods for interacting with the system.
+
+Similarly, :doc:`Database structure <database-erd-ref>`_ is considered to be an internal implementation
+detail of Apache Airflow and you should not assume the structure is going to be maintained in
+backwards-compatible way.
+
+Is Airflow DB part of the Public API of Airflow?
+================================================
+
+The :doc:`Airflow DB <database-erd-ref>`_ is not considered to be part of the public API of Apache Airflow.
+The Airflow DB is the internal database used by Airflow to store information about DAGs, tasks, and
+other aspects of the system's state. This database is not intended to be accessed directly by
+external developers, and the specific details of its schema and structure are not considered
+to be part of the public API.
+
+Instead, developers who want to interact with the Airflow DB should use the :doc:`stable-rest-api-ref` and
+the :doc:`stable-cli-ref`, both providing a number of different interfaces and methods for accessing and
+manipulating data in the database. These APIs are designed to be more stable and user-friendly than
+accessing the Airflow DB directly, and provide a more consistent and predictable way of interacting
+with the system.
+
+Is Stable REST API part of the Public API of Airflow?
+=====================================================
+
+The :doc:`Stable REST API <stable-cli-ref>`_ is considered to be part of the public API of Apache Airflow.
+The REST API is a set of programming interfaces that allow developers to access certain features of
+Airflow over the HTTP protocol, using a standard set of RESTful (representational state transfer) conventions.
+
+The stable REST API is considered to be part of the public API because it is designed to be used by
+external developers to interact with Airflow in a consistent and predictable way. The REST API provides
+access to many of the same features and functionality that are available through the other public APIs
+of Airflow, such as the ability to create and manage DAGs and tasks, and to monitor their status.
+
+Overall, the stable REST API is an important part of the public API of Apache Airflow, and can
+be a useful tool for developers who want to integrate Airflow with other systems or
+build custom tools and applications on top of Airflow.
```

Comment:
Instead of long stories, would keep it concise, and create a bullet list of items that are (and if needed -- are not) part of the public APIs. That reads much easier than having to distill the correct information from text.

Reply:
Removed it altogether.